R/correlation.R

```diff
@@ -1281,6 +1334,29 @@ Correlation <- function(jaspResults, dataset, options){
 
 ### helpers ----
 ### Utility functions for nonparametric confidence intervals ###
+.NormalApproxConfidenceIntervals <- function(obsCor, n, hypothesis="two.sided", confLevel=0.95){
```

Comment:
This seems quite reasonable for the Pearson's correlation, but I forgot how valid this is for Kendall's tau and Spearman's rho

Reply:
Yeah, the problem is we do not need it for Pearson, but for (especially) Kendall, whose CI computation is slow in larger samples. I am also not sure whether it's very reasonable approximation. I could change it if we decide it's not a great idea...